Bottom: ccb766ca3b62574d75d9742aac5be07d63db907b
Top:    b1b273a74a584bc08f9bc1dfd0eeb0aa6dfbf6a9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:08:41 +0000

Refresh of revert-ripple-out-step-emission-1

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 73a9ee9..8a8d5e0 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -289,7 +289,6 @@ get_contour_coord_n_in_mm (PLINE *contour, int n, double *x, double *y)
   *y = COORD_TO_MM (vertex->point[1]); /* FIXME: PCB's coordinate system has y increasing downwards */
 }
 
-
 static void
 fprint_idlist (FILE *f, int *ids, int num_ids)
 {
